src/main/scala/geotrellis/sdg/FilteredRoadsPipeline.scala

```diff
@@ -0,0 +1,38 @@
+package geotrellis.sdg
+
+import java.net.URI
+
+import geotrellis.layer.LayoutLevel
+import geotrellis.vector._
+import geotrellis.vectortile.{VBool, VString}
+import org.apache.spark.sql.{DataFrame, Row}
+import vectorpipe.vectortile.{LayerMultiplicity, Pipeline, SingleLayer, VectorTileFeature}
+
+case class FilteredRoadsPipeline(geometryColumn: String,
+                                 baseOutputURI: URI,
+                                 reduceToIncludedZoom: Int) extends Pipeline {
+
+  override val layerMultiplicity: LayerMultiplicity = SingleLayer("roads")
+
+  override def reduce(input: DataFrame, layoutLevel: LayoutLevel, keyColumn: String): DataFrame = {
+    if (reduceToIncludedZoom == layoutLevel.zoom) {
+      input.where("isIncluded = true")
+    } else {
+      input
+    }
+  }
```

Comment:
I'm not sure if this filter is going to be good enough for larger countries where the top tiles might be in Megabytes.
Nothing to do now, just look here when this happens for USA or Germany.

Reply:
Good point. I suppose the next steps here would be to do something like filter down to some subset of included roads, such as  "trunk" and "primary" at some static zoom level lower than the current reduceToIncludedZoom breakpoint?